src/store/reducers/transactions.ts

```diff
@@ -41,9 +56,17 @@ function transactionOperations(
 ): TransactionOperationView[] {
   switch (action.type) {
     case getType(loadAccountTransactions):
+    case getType(setActiveAccount):
       return []
     case getType(accountTransactionsLoaded):
       return action.payload.operations
+        .filter(({ id }) => state.every(({ id: existingId }) => id !== existingId))
```

Comment:
I think it'd be nice to put this logic in a separate function outside this case statement, to help remove the clutter and repetition, making it easier to grasp what's going on in a quick look. eg.

```typescript
uniqueIdInList = (list) => ({ id }) => list.every(({ id: existingId }) => id !== existingId)
```
which could then be called with something like this
```typescript
case getType(...):
  return action.payload.operations
    .filter(uniqueIdInList(state))
    .concat(state)
```

Reply:
Yes, I contemplated to create a function but decided not to pollute the reducer file.
done